src/components/ProfilePage/Name.jsx

```diff
@@ -72,19 +71,19 @@ class Name extends React.Component {
                   such to fully get rid of it.
                   */}
                   <p className="h5">{name}</p>
-                  <FormText id={`${formId}-help-text`}>
+                  <small className="form-text text-muted" id={`${formId}-help-text`}>
```

Comment:
So this one is interesting, as it's sorta difficult for folks to know we use `small` to represent form helper text.  That said, it's what they do in the bootstrap documentation.  I guess if we had our own FormRow/FormGroup component that'd help with this.  &shrug;

Reply:
Yea I think a higher level component would manage this. Also, it seems weird to have small here. I thought it was an inline element by default. I would rather moving forward do `<p className="form-text text-muted small">stuff</p>`. Or just update the style for form-text to include the others